components/molecules/tabs/components/tabs.js

```diff
@@ -52,12 +52,16 @@ export const handleScroll = (el, grabWalkSpeed, grabTimeout) => {
 }
 
 export const Tabs = ({
+  centerTabs,
   children,
   className,
+  indicatorSize,
+  gap,
```

Comment:
Are these props that you've added enough for our needs? As far as I remember, we talked about two props `indicator/hasIndicator/withIndicator` and `border/hasBorder/withBorder` so that we can have three different variants of tabs.
Are these changes going to cover our requirements on IUI website?